ui/components/multichain/import-tokens-modal/import-tokens-modal.js

```diff
@@ -39,11 +39,11 @@ import {
   ButtonPrimary,
   FormTextField,
   Modal,
-  ModalContent,
-  ModalHeader,
   ModalOverlay,
   Text,
 } from '../../component-library';
+import { ModalContent } from '../../component-library/modal-content/deprecated';
+import { ModalHeader } from '../../component-library/modal-header/deprecated';
 import TokenSearch from '../../app/import-token/token-search';
 import TokenList from '../../app/import-token/token-list';
 
```

Comment:
The screenshots and screencast below show an issue with the `Tabs` component and the tab menu having `position: fixed;` styles. I've created an [issue here](https://github.com/MetaMask/metamask-extension/issues/22224) to address 

### Before
Storybook shows existing issue with tabs component in the import tokens modal

https://github.com/MetaMask/metamask-extension/assets/8112138/64270b03-1638-49ea-81f0-2900956df8d7

### After
Shows updates made with the new Modal components

https://github.com/MetaMask/metamask-extension/assets/8112138/d36f4aa2-6ab8-4c13-91df-9be3ab99ac31

https://github.com/MetaMask/metamask-extension/assets/8112138/0151786c-733c-4c55-af01-435434814df5